jee-learning-platform/backend/app/routes/content.py

```
@@ -0,0 +1,318 @@
+from typing import List
+from fastapi import APIRouter, Depends, HTTPException, status
```

Ruff (0.11.9)

`fastapi.status` imported but unused

Remove unused import: `fastapi.status`

(F401)